cosmos/urls.py

```diff
@@ -13,11 +13,12 @@
 admin.autodiscover()
 
 urlpatterns = [
-    path("", include("apps.users.urls")),
-    path("", include("apps.legacy.urls")),
-    re_path(r"^sitemap\.xml$", sitemap, {"sitemaps": {"cmspages": CMSSitemap}}),
     path("admin/", admin.site.urls),
+    path("accounts/", include("django.contrib.auth.urls")),
+    path("accounts/", include("apps.users.urls")),
+    path("accounts/", include("apps.legacy.urls")),
```

Comment:
is there a way to make the redirects defined in views.py dynamic (prefixes not hard-coded)?

Reply:
What exactly do you mean with that, and what would be the benefit of not hardcoding the prefixes? Cause we could create a variable there and use that, but i feel that would reduce clarity for very little advantage in being able to quickly modify sth. Especially as the url should unless absolutely needed, never be changed. And as these prefixes are only ever used in this file, i dont think it will be too big of an issue having them hardcoded.

Reply:
nvm, I was looking at `apps/users/views.py` where it referred to a static file `render(request, "user/register.html", {"user_form": user_form, "profile_form": profile_form})` but misunderstood that it wasn't a redirect link. I was looking for what `reverse()` was already doing in other files, my bad. I'll change my approval state.